modules/dnn/test/test_backends.cpp

```diff
@@ -78,17 +78,6 @@ class DNNTestNetwork : public DNNTestLayer
     {
         if (outputLayer == "detection_out")
         {
-            if (backend == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019)
```

Comment:
Can you please check why nGraph path works correctly this way? I thought outputs from both APIs should be the same.

Reply:
It strange, but nGraph works in both cases